packages/next/src/server/lib/router-utils/setup-dev.ts

```diff
@@ -311,20 +314,18 @@ async function startWatcher(opts: SetupOpts) {
     async function processResult(
       result: TurbopackResult<WrittenEndpoint>
     ): Promise<TurbopackResult<WrittenEndpoint>> {
-      for (const file of result.serverPaths
-        .map((p) => path.join(distDir, p))
-        .concat([
-          // We need to clear the chunk cache in react
-          require.resolve(
-            'next/dist/compiled/react-server-dom-webpack/cjs/react-server-dom-webpack-client.edge.development.js'
```

Comment:
This is no longer cleared?

Could we add that to `packages/next/src/build/webpack/plugins/nextjs-require-cache-hot-reloader.ts`? Or isn't it necessary?